canonical-web-and-design/ubuntu.com
# Add 'New' label to 'Account users' navigation link

## Done

When user has not dismissed the notification show on /advantage about
account management show the 'New' label on 'Account users' navigation link.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage?test_backend=true
- Make sure you see the `New` label on the secondary navigation before `Account Users`
- Dismiss the notification about managing users on the page
- Refresh and make sure the label and the notification are gone.
